python/lsst/ts/mtmount/mtmount_commander.py

```diff
@@ -77,15 +79,19 @@ def __init__(self, enable):
             "encoder",
             "generalPurposeGlycolWater",
             "mountControlMainCabinet",
+            "oilSupplySystem",
         ):
-            topic = getattr(self.remote, f"tel_{topic_name}")
+            topic = getattr(self.remote, f"tel_{telemetry_topic_name}")
             topic.callback = None
-        # TODO 37114: once ts_xml 15 is released,
-        # use the above loop to block oilSupplySubsystem
-        for topic_name in ("oSS", "oilSupplySubsystem"):
-            topic = getattr(self.remote, f"tel_{topic_name}", None)
-            if topic is not None:
-                topic.callback = None
+
+        # Treat a few events like telemetry, to reduce chatter.
```

Comment:
Out of curiosity, was this introduced to prevent having to change ts_xml or for another reason?

Reply:
The point was that instead of publishing every event, even if the data does not change, only print it if the data has changed. A steady stream of clockOffset events tends to fill up the screen with uninteresting data. Thus "reduce chatter".

Reply:
That's good to know. My question was weather you plan to transform this into telemetry permanently or wanna stick with this workaround.

Reply:
I intend to keep this long term.